newIDE/app/src/Profile/Achievement/AchievementList.js

```diff
@@ -0,0 +1,90 @@
+// @flow
+import React, { useEffect, useState } from 'react';
+import { DateFormat } from '@lingui/macro';
+
+import { Column, Line } from '../../UI/Grid';
+import Text from '../../UI/Text';
+import {
+  type Badge,
+  type Achievement,
+} from '../../Utils/GDevelopServices/User';
+
+type Props = {|
+  badges: Array<Badge>,
+  achievements: Array<Achievement>,
+|};
+
+const achievementBaseline = {
+  width: '100%',
+  padding: '0 20px',
+};
+
+const styles = {
+  achievementsContainer: {
+    maxHeight: 250,
+    overflowY: 'scroll',
+  },
+  unlockedAchievement: {
+    ...achievementBaseline,
+    color: 'green',
+    display: 'flex',
+    justifyContent: 'space-between',
```

Comment:
This could maybe be a `Line` with justifyContent="space-between".
In general I tend to do all the layouting with Line/Column/Spacer/ScrollView as much as possible.